shell/platform/darwin/ios/platform_service_ios.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_IOS_PLATFORM_SERVICE_IOS_H_
+#define FLUTTER_SHELL_PLATFORM_DARWIN_IOS_PLATFORM_SERVICE_IOS_H_
+
+#include "flutter/shell/common/platform_service.h"
+
+namespace shell {
+
+class PlatformServiceIOS : public PlatformService {
```

Comment:
There is very little iOS specific functionality here. This should probably go in `shell/platform/darwin/common` so that it can also be used by the desktop target for free.